Cargo.toml

```diff
@@ -6,6 +6,7 @@ members = [
   "helix-tui",
   "helix-syntax",
   "helix-lsp",
+  "helix-dap"
```

Comment:
I don't know a thing about anything going on in this PR, but I will say I think this should have a trailing comma
```
  "helix-dap",
```

Reply:
Definitely!

Reply:
```suggestion
  "helix-dap",
```